test/torch/model/test_deepar.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Maybe add a simple accuracy test as well (on constant or so)?

Reply:
Yes, that I’ll add. I also wanted to run a few training/evaluation jobs to compare with the MXNet implementation on some dataset, just to get a feeling for any apparent discrepancy: I’ll post results here as soon as I do